cities/cities.json

```diff
@@ -0,0 +1,54 @@
+[  
```

Comment:
I would prefer a hashmap (`id -> options`) instead of a list. That would automatically ensure that IDs are unique. Since we don't care about order we don't loose anything.